apps/docs/netsim/README.md

```diff
@@ -0,0 +1,218 @@
+# Internet Simulator
+
+The Internet Simulator is an app running on studio.code.org that simulates a 
+network for a group of users.  It does not actually establish peer-to-peer 
+connections between clients; instead, each client communicates with the 
+standard [app storage API](../../docs/client-api.md) on Code.org's servers, so 
+all clients interact with a common network state hosted with us.  Beyond that 
+network state, there isn't any intelligence on the server - clients are 
+trusted, and distribute work for running the simulation among themselves 
+according to some simple rules.
+
+* [Shared Storage](#shared-storage)
+  * [Shards](#shards)
+  * [Tables](#tables)
+* [Distribution of Responsibility](#distribution-of-responsibility)
+  * [Router Status](#router-status)
+  * [Router Connection Limits](#router-connection-limits)
+  * [Address Assignment](#address-assignment)
+  * [Message Routing](#message-routing)
+  * [Shard Cleaning](#shard-cleaning)
+
+## Shared Storage
+
+The network simulator has its own app ID within the app storage system, and creates/interacts with shared tables to model a common network state.
```

Comment:
Could you make the wrapping on this line consistent with other paragraphs? (Note, I explicitly don't mind that tables are wider than 80 characters.)